CONTRIBUTING.md

```diff
@@ -17,27 +17,53 @@ Cloudant-client is written in Java and uses maven as its build tool.
 
 Follow the instructions for your platform.
 
-### Maven
-
-Using brew
+### Gradle
 
-```bash
-$ brew install maven
-```
+The project uses the gradle wrapper to download  specified version of gradle.
 
 # Building the library
 
 The project should build out of the box with:
 
 ```bash
-$ mvn compile
+$ ./gradlew compileJava
 ```
+Note: on windows the command to run is gradlew.bat rather than gradlew
 
 ### Running the tests
 
-After configuring using cloudant-sample.properties and
-cloudant-2.sample.properties as guides, run:
+The tests can be configured before running, by default they use the local
+CouchDB. To run tests with a remote CouchDB, you need set the details of this
+CouchDB server, including access credentials: add the following to
+`gradle.properties` in the same folder as `build.gradle`:
+```
+systemProp.test.couch.username=yourUsername
+systemProp.test.couch.password=yourPassword
+systemProp.test.couch.host=couchdbHost # default localhost
+systemProptest.couch.port=couchdbPort # default 5984
+systemProp.test.couch.http=[http|https] # default 5984
+```
+or you can use can provide a URL containing all the above information:
+```
+systemProp.test.couch.uri
+```
```

Comment:
Provide an example and note whether any parts can be left out (e.g., is port required?)